blueprints/zesty-deploy-config/files/.circleci/config.yml

```diff
@@ -0,0 +1,123 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:6.14-browsers
+    environment:
+      - TZ: "/usr/share/zoneinfo/America/Los_Angeles"
+    steps:
+        - checkout
+
+        # Install global dependencies
+        - run:
+            name: Install global dependencies
+            command: |
+              sudo npm install -g yarn@rc
+              <% if (bowerRequired) { %>sudo npm install -g bower<% } %>
+
+        # Install yarn dependencies
+        # Attempt to restore from cache. If not install dependencies and save.
+        - restore_cache:
+            keys:
+              - v1-yarn-deps-{{ checksum "yarn.lock" }}
+        - run:
+            name: Install yarn dependencies
+            command: yarn install --frozen-lockfile
+        - run:
+            name: Rebuild Node
+            command: npm rebuild node-sass
+        - save_cache:
+            key: v1-yarn-deps-{{ checksum "yarn.lock" }}
+            paths:
+              - "./node_modules"
+
+        <% if (bowerRequired) { %>
```

Comment:
I'm seeing an error on the circle builds around this:

https://circleci.com/gh/zestyzesty/shifts-app/334

Can you tell what's going on with that?

Reply:
That's actually a test run with the an older version (whilst this was in development).